internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -290,6 +293,9 @@ func (r *ISBServiceRolloutReconciler) reconcile(ctx context.Context, isbServiceR
 		if err != nil {
 			return ctrl.Result{}, fmt.Errorf("error getting ISBService: %v", err)
 		}
+		if err = r.applyPodDisruptionBudget(ctx, existingISBServiceDef); err != nil {
+			return ctrl.Result{}, fmt.Errorf("failed to apply PodDisruptionBudget for ISBService %s, UID=%s, err: %v", existingISBServiceDef.GetName(), existingISBServiceDef.GetUID(), err)
+		}
```

Comment:
when we already have an existing "promoted" isbsvc, make sure corresponding PDB exists and if not, create it (auto healing)